packages/logger/tests/unit/setPowertoolsLogData.test.ts

```diff
@@ -0,0 +1,38 @@
+import { Logger } from '../../src/Logger.js';
+import { describe, it, expect, vi } from 'vitest';
+
+describe('Logger PowertoolsLogData AWS_REGION', () => {
+  it.each([
+    [undefined, ''],
+    ['us-west-2', 'us-west-2'],
+  ])('should set awsRegion to %s when AWS_REGION env var is %s', (envValue, expected) => {
+    vi.stubEnv('AWS_REGION', envValue);
+    const logger = new Logger();
+    expect(logger['powertoolsLogData'].awsRegion).toBe(expected);
+    vi.unstubAllEnvs();
```

Comment:
We can move these to a `beforeEach` bloc ks o we don't have to repeatedly call `vi.unstubAllEnvs();`